exchange_server/assets/dashboards/overview.json

```diff
@@ -0,0 +1,755 @@
+            "id": 18
+        },
+        {
+            "title_size": 16,
+            "title": true,
+            "title_align": "left",
+            "title_text": "I/O operations rate",
+            "height": 13,
+            "tile_def": {
+                "viz": "timeseries",
+                "requests": [
+                    {
+                        "q": "avg:exchange.database.io_db_reads_attached_persec{*}",
+                        "style": {
+                            "width": "normal",
+                            "palette": "dog_classic",
+                            "type": "solid"
+                        },
+                        "type": "line"
+                    },
+                    {
+                        "q": "avg:exchange.database.io_db_writes_attached_persec{*}",
+                        "style": {
+                            "width": "normal",
+                            "palette": "purple",
+                            "type": "solid"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "yaxis": {
+                    "max": "auto",
+                    "scale": "linear",
+                    "includeZero": true,
+                    "label": "",
+                    "min": "auto"
+                }
+            },
+            "width": 37,
+            "legend": false,
+            "y": 26,
+            "x": 112,
+            "type": "timeseries",
+            "id": 19
+        },
+        {
+            "title_size": 16,
+            "title": true,
+            "title_align": "left",
+            "title_text": "CPU usage",
+            "height": 13,
+            "tile_def": {
+                "viz": "timeseries",
+                "requests": [
+                    {
+                        "q": "avg:exchange.processor.cpu_time{*}",
+                        "style": {
+                            "width": "normal",
+                            "palette": "dog_classic",
+                            "type": "solid"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "yaxis": {
+                    "max": "auto",
+                    "scale": "linear",
+                    "includeZero": true,
+                    "label": "",
+                    "min": "auto"
+                }
+            },
+            "width": 37,
+            "legend": false,
+            "y": 43,
+            "x": 1,
+            "type": "timeseries",
+            "id": 20
+        },
+        {
+            "title_size": 16,
+            "title": true,
+            "title_align": "left",
+            "title_text": "Active tasks",
+            "height": 7,
+            "tile_def": {
+                "viz": "query_value",
+                "requests": [
+                    {
+                        "q": "sum:exchange.workload_management.active_tasks{*}",
```

Comment:
Same comment, I think this would look nicer with a timeseries graph split by instance